sdk/eventhub/Azure.Messaging.EventHubs/src/Diagnostics/EventDataDiagnosticsExtensions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+
+namespace Azure.Messaging.EventHubs.Diagnostics
+{
+    /// <summary>
+    ///   Diagnostic extension methods for <see cref="EventData"/>.
+    /// </summary>
+    ///
+    internal static class EventDataDiagnosticsExtensions
```

Comment:
Should we do these as a dedicated diagnostics type, or as a general `EventDataExtensions` type where we can add additional things in the future?  I'm kind of on the fence, but I don't *think* we'll have a bunch of other diagnostics-focused extensions... do you?

Reply:
I don't think so. In Track One this was the only diagnositcs-focused Event Data extension method and I think we won't be adding anything for now. I'll update the class and file names.